### PR TITLE
Projector.service: Topic slider

### PR DESCRIPTION
**Pull request by r-peschke**

### CODE REVIEW

File: internal/projector/slide/topic.go [hunk 1]
```diff
@@ -30,7 +32,50 @@ func topicFromMap(in map[string]json.RawMessage) (*dbTopic, error) {
...
+
+		topic, err := topicFromMap(data)
+		if err != nil {
+			return nil, nil, fmt.Errorf("get topic: %w", err)
+		}
+
+		var itemNumber string
+		if topic.AgendaItemID > 0 {
+			itemNumber = fetch.String(ctx, fmt.Sprintf("agenda_item/%d/item_number", topic.AgendaItemID))
+		} else {
```
Comment (ostcar) [inline internal/projector/slide/topic.go:61]: You can remove the else block.

Go sets the variable "itemNumber" to an empty string. So the else block is a noop

File: internal/projector/slide/topic.go [hunk 2]
```diff
@@ -30,7 +32,50 @@ func topicFromMap(in map[string]json.RawMessage) (*dbTopic, error) {
...
+		)
+
+		topic, err := topicFromMap(data)
+		if err != nil {
+			return nil, nil, fmt.Errorf("get topic: %w", err)
+		}
+
+		var itemNumber string
+		if topic.AgendaItemID > 0 {
+			itemNumber = fetch.String(ctx, fmt.Sprintf("agenda_item/%d/item_number", topic.AgendaItemID))
```
Comment (ostcar) [inline internal/projector/slide/topic.go:60]: You do not need the fmt.Sprintf here. You can just call:

```go
fetch.String(ctx, "agenda_item/%d/item_number", topic.AgendaItemID)
```

**Review by ostcar — CHANGES_REQUESTED** [review]

File: internal/projector/slide/topic.go (hunk 1 above)
Comment (r-peschke) [reply internal/projector/slide/topic.go:61]: done

File: internal/projector/slide/topic.go (hunk 2 above)
Comment (r-peschke) [reply internal/projector/slide/topic.go]: done

**Review by ostcar — APPROVED** [review]

**Pull request merged**